inc/common/pjrt_implementation/buffer_instance.h

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+#include "common/pjrt_implementation/event_instance.h"
+#include "common/status.h"
+#include "tt/runtime/runtime.h"
+#include "xla/pjrt/c/pjrt_c_api.h"
```

Comment:
I'd expect that lot more headers need to include this, `bindings.h` for example. Let's be explicit about includes (which we are not forwarding).

Also, this is a third party include so it needs to be in a separate group.

Reply:
You left this one unresolved, maybe my comment was a bit unclear. I was referring to `xla/pjrt/c/pjrt_c_api.h` include. It is a third party include, so it should be in a separate group, similar to `tt/runtime/runtime.h`. Structures from that file are used in most of the pjrt_imlementation .h files so you should explicitly include it in those files instead of relying on transitive includes. For example `api_bindings.h` is using `PJRT_Api` which is defined in `xla/pjrt/c/pjrt_c_api.h`, which you are not including there, but your build is passing because of transitive include through `common/pjrt_implementation/error_instance.h`.